controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java

```diff
@@ -220,7 +220,7 @@ public void initializeStreamWriters(final EventStreamClientFactory clientFactory
                                                                                   KeyValueTableConfiguration kvtConfig,
                                                                                   final long createTimestamp,
                                                                                   Executor executor) {
-        return eventHelper.checkDone(() -> isCreated(scope, kvtName, kvtConfig, executor))
```

Comment:
same here -- comparing the kvt config will determine if the create was actually performed with the supplied config or a different one from a concurrent request that got executed first.

Reply:
Configuration is being checked in isCreateProcessed() method. It is not required in isCreated() method and was anyways passed but not used.